contrib/html_demo/UICode/references.txt

```diff
@@ -0,0 +1,28 @@
+<!--
+    Frameworks and examples used to build this code (not sure how to reference these): 
```

Comment:
Best to mention these in the readme.md, rather than in a separate file. Only mention urls where you copied code from, urls you just used to ramp up on the technology don't need to be listed (except there is one which you really liked and think is great). E.g. the stackoverflow urls can probably mostly/all get deleted etc.

Reply:
Removed